plugins/hydra_ray_launcher/tests/test_ray_launcher.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import pytest
+
+from hydra.test_utils.launcher_common_tests import (
+    IntegrationTestSuite,
+    LauncherTestSuite,
+)
+
+# noinspection PyUnresolvedReferences
+from hydra.test_utils.test_utils import sweep_runner  # noqa: F401
+
+
+@pytest.mark.parametrize("launcher_name, overrides", [("ray", [])])
+class TestRayLauncher(LauncherTestSuite):
+    pass
+
+
+@pytest.mark.parametrize(
+    "task_launcher_cfg, extra_flags, plugin_module",
+    [
+        (
+            {
+                "defaults": [
+                    {"hydra/launcher": "ray"},
+                    {"hydra/hydra_logging": "hydra_debug"},
+                    {"hydra/job_logging": "disabled"},
+                ]
+            },
+            ["-m"],
+            "hydra._internal.core_plugins.basic_launcher",
+        )
+    ],
+)
+class TestRayLauncherIntegration(IntegrationTestSuite):
+    """
+    Run this launcher through the integration test suite.
+    """
+
+    pass
```

Comment:
I took this from the example launcher plugin tests. Not exactly sure what to test other than this.

Reply:
This is good.
Ray is running locally by default, right?